templates/helm/orchestration-workflows/clinvar/templates/workflow.yaml

```diff
@@ -35,7 +35,7 @@ spec:
   resetValues: true
   chart:
     git: git://github.com/databiosphere/clinvar-ingest.git
-    ref: master
+    ref: ra-DSPDC-1009-no-rows
```

Comment:
TODO: Revert back to `master`

Reply:
should be good to revert now, is merged!